ci/github-actions.md

```diff
@@ -41,6 +41,22 @@ jobs:
 
 Due to the `on` block, this workflow will run the `test` job on all commits to `master` and all commits to pull requests that have been opened against `master`.
 
+For most of our apps deployed to Heroku, we make use of local `.env` and `.env.example` files to store our secrets. If your app is setup this way, add these lines to the `test` block of `main.yml`
```

Comment:
Rather than framing this in a copy/paste sort of way, can we say "Add a step to create a .env file from your example .env file, prior to running your tests"? It might seem like a small change, but I think it helps to set expectations for these docs, which is that we're trying to cover the fundamentals and equip readers with the vocabulary to expand upon these basic examples.

Reply:
```suggestion
If your tests need any additional configurations, such as a `.env` file or a local settings file, add a step to your `test` job to create or copy the necessary files, prior to running the tests. For example:
```

Reply:
(Another benefit of this framing is that it is decoupled from our standard approach, that way if the approach changes, we don't need to remember to update these docs along with it.)

Reply:
cool. accepted the change